lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -47,11 +52,11 @@ export default function AssignmentActivity() {
             />
           </div>
         )}
-        <h2 data-testid="title" className="text-lg text-brand font-semibold">
+        <CardTitle tagName="h2" data-testid="title">
```

Comment:
I moved away from `CardTitle` on a previous PR, but it was actually not needed, so going back to it to have implicit styles.